tests/integration.rs

```diff
@@ -85,21 +71,20 @@ impl io::Read for KmsConnection {
 }
 
 /// Receives incoming KMS connection then sends commands
-struct KmsDevice {
```

Comment:
Somewhat superfluous name change, but in debugging this I was confused what `KmsDevice` was and discovered it was the subprocess under test.